Gemfile

```diff
@@ -78,7 +78,7 @@ gem 'wisper', '2.0.0'
 # TODO: bump up gem to 2.0
 gem 'facebook-messenger'
 gem 'line-bot-api'
-gem 'twilio-ruby', '~> 5.32.0'
+gem 'twilio-ruby', '~> 5.65'
```

Comment:
We need to update this library to get the `use_inbound_webhook_on_number` option in `app/services/twilio/webhook_setup_service.rb`.